rootfs/usr/lib/systemd/system-shutdown/mt7921e.shutdown

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+# Unload the mt7921e driver to avoid kernel panic on reboot.
+rmmod mt7921e
```

Comment:
Should it be `/usr/bin/rmmod` ? At least that is what I see (full paths) being used in other scripts.

Reply:
Good catch. I used the sbin path as it requires root privileges.